client/state/ui/editor/selectors.js

```diff
@@ -51,6 +52,20 @@ export function getEditorDuplicatePostPath( state, siteId, postId, type = 'post'
 	return `${ editorNewPostPath }?copy=${ postId }`;
 }
 
+/**
+ * Returns the editor URL of a Calypsoified site for duplicating a given site ID, post ID pair.
+ *
+ * @param  {Object} state       Global state tree
+ * @param  {Number} siteId      Site ID
+ * @param  {Number} postId      Post ID
+ * @param  {String} type        Post type
+ * @return {String}             Editor URL path
+ */
+export function getCalypsoifyEditorDuplicatePostPath( state, siteId, postId, type = 'post' ) {
+	const gutenbergEditorUrl = getGutenbergEditorUrl( state, siteId, null, type );
```

Comment:
What if we updated the signature of `getGutenbergEditorUrl` to `getGutenbergEditorUrl( state, siteId, postId, type, queryParams={} )`